docs/guides/getting-started-istio.md

```diff
@@ -0,0 +1,214 @@
+## Running Kong Ingress Controller with Istio
+
+By following this guide, you will:
+* install Istio v1.6.7 and Kong in your cluster,
+* deploy an example Istio-enabled application (_bookinfo_),
+* deploy an `Ingress` customized with a `KongPlugin` for the example application,
+* make several requests to the sample application via Kong and Istio,
+* see the performance metrics of the sample application, provided by Istio.
```

Comment:
```suggestion
In this guide, you will:
* Install Istio v1.6.7 and Kong in your cluster.
* Deploy an example Istio-enabled application (_bookinfo_).
* Deploy an `Ingress` customized with a `KongPlugin` for the example application.
* Make several requests to the sample application via Kong and Istio.
* See the performance metrics of the sample application, provided by Istio.
```
See https://docs.microsoft.com/en-us/style-guide/scannable-content/lists

Why the convention exists I don't really know, but normally the bullets are treated as quasi-independent sentences and are styled as such.